zio-schema-protobuf/shared/src/main/scala/zio/schema/codec/ProtobufCodec.scala

```diff
@@ -455,7 +454,7 @@ object ProtobufCodec extends Codec {
         case (wt, fieldNumber) if fieldNumber <= cases.length =>
           val subtypeCase = cases(fieldNumber - 1)
           wt match {
-            case LengthDelimited(width) =>
+            case WireType.LengthDelimited(width) =>
```

Comment:
For some reason, `import zio.schema.codec.ProtobufCodec.Protobuf.WireType.LengthDelimited` wasn't being picked up for me, so I explicitly changed these to `WireType.LengthDelimited`, which seems to be pretty common in other parts of the file.